src/assets/TokenListController.ts

```diff
@@ -1,49 +1,43 @@
-import contractMap from '@metamask/contract-metadata';
 import type { Patch } from 'immer';
 import { Mutex } from 'async-mutex';
 // eslint-disable-next-line import/no-named-as-default
 import AbortController from 'abort-controller';
 import { BaseController } from '../BaseControllerV2';
 import type { RestrictedControllerMessenger } from '../ControllerMessenger';
-import { safelyExecute } from '../util';
-import { fetchTokenList, fetchTokenMetadata } from '../apis/token-service';
+import { safelyExecute, isTokenDetectionEnabledForNetwork } from '../util';
+import { fetchTokenList } from '../apis/token-service';
 import { NetworkState } from '../network/NetworkController';
-import { PreferencesState } from '../user/PreferencesController';
+import {
+  AggregatorKey,
+  formatAggregatorNames,
+  formatIconUrlWithProxy,
+} from './assetsUtil';
 
 const DEFAULT_INTERVAL = 60 * 60 * 1000;
 const DEFAULT_THRESHOLD = 60 * 30 * 1000;
 
 const name = 'TokenListController';
 
-type BaseToken = {
+export type RawToken = {
```

Comment:
What is a "raw" token? Are there other kinds of tokens that differ from this?

Reply:
As a followup, is it possible to rename this `Token` or is that too generic?

Reply:
I've renamed RawToken to be TokenListToken for now since it's a token that comes from the TokenList. I didn't want to change or use other Token types yet since I wanted to keep the scope small. In the future when we consolidate all of the types, I'd like to remove all but one.